requirements/test.txt

```diff
@@ -172,7 +172,7 @@ pylint-plugin-utils==0.6
     #   -r requirements/base.txt
     #   pylint-celery
     #   pylint-django
-pyparsing==3.0.1
+pyparsing==2.4.7
```

Comment:
`packaging==21.2` constrained the `pyparsing<3.0.0` in this [PR](https://github.com/pypa/packaging/pull/471) which caused the dowgrading of this package.